identifiers/country-de/aliases.csv

```diff
@@ -15,603 +15,603 @@ ocd-division/country:de/land:sl,ocd-division/country:de/state:sl,Added for backw
+ocd-division/country:de/state:rp/cd:197,ocd-division/country:de/ed:neuwied,Neuwied,,
+ocd-division/country:de/state:rp/cd:198,ocd-division/country:de/ed:ahrweiler,Ahrweiler,,
+ocd-division/country:de/state:rp/cd:199,ocd-division/country:de/ed:koblenz,Koblenz,,
+ocd-division/country:de/state:rp/cd:200,ocd-division/country:de/ed:mosel~rhein-hunsrück,Mosel/Rhein-Hunsrück,,
+ocd-division/country:de/state:rp/cd:201,ocd-division/country:de/ed:kreuznach,Kreuznach,,
+ocd-division/country:de/state:rp/cd:202,ocd-division/country:de/ed:bitburg,Bitburg,,
+ocd-division/country:de/state:rp/cd:203,ocd-division/country:de/ed:trier,Trier,,
+ocd-division/country:de/state:rp/cd:204,ocd-division/country:de/ed:montabaur,Montabaur,,
+ocd-division/country:de/state:rp/cd:205,ocd-division/country:de/ed:mainz,Mainz,,
+ocd-division/country:de/state:rp/cd:206,ocd-division/country:de/ed:worms,Worms,,
+ocd-division/country:de/state:rp/cd:207,ocd-division/country:de/ed:ludwigshafen~frankenthal,Ludwigshafen/Frankenthal,,
+ocd-division/country:de/state:rp/cd:208,ocd-division/country:de/ed:neustadt_-_speyer,Neustadt - Speyer,,
+ocd-division/country:de/state:rp/cd:209,ocd-division/country:de/ed:kaiserslautern,Kaiserslautern,,
+ocd-division/country:de/state:rp/cd:210,ocd-division/country:de/ed:pirmasens,Pirmasens,,
+ocd-division/country:de/state:rp/cd:211,ocd-division/country:de/ed:südpfalz,Südpfalz,,
+ocd-division/country:de/state:sh/cd:1,ocd-division/country:de/ed:flensburg_-_schleswig,Flensburg - Schleswig,,
+ocd-division/country:de/state:sh/cd:10,ocd-division/country:de/ed:herzogtum_lauenburg_-_stormarn-süd,Herzogtum Lauenburg - Stormarn-Süd,,
+ocd-division/country:de/state:sh/cd:11,ocd-division/country:de/ed:lübeck,Lübeck,,
+ocd-division/country:de/state:sh/cd:2,ocd-division/country:de/ed:nordfriesland_-_dithmarschen_nord,Nordfriesland - Dithmarschen Nord,,
+ocd-division/country:de/state:sh/cd:3,ocd-division/country:de/ed:steinburg_-_dithmarschen_süd,Steinburg - Dithmarschen Süd,,
+ocd-division/country:de/state:sh/cd:4,ocd-division/country:de/ed:rendsburg-eckernförde,Rendsburg-Eckernförde,,
+ocd-division/country:de/state:sh/cd:5,ocd-division/country:de/ed:kiel,Kiel,,
+ocd-division/country:de/state:sh/cd:6,ocd-division/country:de/ed:plön_-_neumünster,Plön - Neumünster,,
+ocd-division/country:de/state:sh/cd:7,ocd-division/country:de/ed:pinneberg,Pinneberg,,
+ocd-division/country:de/state:sh/cd:8,ocd-division/country:de/ed:segeberg_-_stormarn-mitte,Segeberg - Stormarn-Mitte,,
+ocd-division/country:de/state:sh/cd:9,ocd-division/country:de/ed:ostholstein_-_stormarn-nord,Ostholstein - Stormarn-Nord,,
+ocd-division/country:de/state:sl/cd:296,ocd-division/country:de/ed:saarbrücken,Saarbrücken,,
+ocd-division/country:de/state:sl/cd:297,ocd-division/country:de/ed:saarlouis,Saarlouis,,
+ocd-division/country:de/state:sl/cd:298,ocd-division/country:de/ed:st._wendel,St. Wendel,,
+ocd-division/country:de/state:sl/cd:299,ocd-division/country:de/ed:homburg,Homburg,,
```

Comment:
We should _also_ have the mapping from the `ed:<number>` to `ed:<name>` in aliases. So for example we should have `ocd-division/country:de/ed:299` equivalent to `ocd-division/country:de/ed:homburg`.

Or are these left out because of the ID shifts that happened with the latest changes by the government?

Reply:
+1

Reply:
So adding more mappings, so would have 3 aliases for `ocd-division/country:de/ed:homburg`?

```
ocd-division/country:de/land:sl/wahlkreis:homburg,ocd-division/country:de/ed:homburg
ocd-division/country:de/state:sl/cd:299,ocd-division/country:de/ed:homburg
ocd-division/country:de/ed:299,ocd-division/country:de/ed:homburg
```

Reply:
Correct, the idea being that data using any of the old formats can be translated to the latest OCD IDs by using the mappings from the aliases file

Reply:
- added alias entries for all numeric based IDs pointing to the name based ones